nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -476,14 +477,16 @@ define([
         }
 
         function handleRunStatus(message) {
-            busEventManager.remove(runStatusListener);
+            resetRunStatusListener();
             switch (message.event) {
                 case 'launched_job_batch':
                     // remove any existing jobs
                     jobManager.initBatchJob(message);
+                    if (cancelBatch) {
+                        cancelBatchJob();
+                        break;
+                    }
```

Comment:
for the edge case where the cancel request comes in before the list of jobs started is sent by the backend